### PR TITLE
Generate DefaultListCount

### PR DESCRIPTION
**Pull request by Ramky-Infoblox**

### CODE REVIEW

File: example/feature_demo/demo_service.pb.gorm.go [hunk 1]
```diff
@@ -585,6 +585,58 @@ type IntPointORMWithAfterListFind interface {
...
+			return nil, err
+		}
+		pbResponse = append(pbResponse, &temp)
+	}
+	pi := &query1.PageInfo{TotalSize: total}
+	pbPgResponse := &IntPointListResponse{pbResponse, pi}
+	return pbPgResponse, nil
+}
+
+type IntPointListResponse struct {
```
Comment (chinmayb) [inline example/feature_demo/demo_service.pb.gorm.go:635]: what does IntPoint mean ?

Comment (Ramky-Infoblox) [reply example/feature_demo/demo_service.pb.gorm.go:635]: It generated from existing example demo_service.proto file for test purpose

Comment (chinmayb) [reply example/feature_demo/demo_service.pb.gorm.go:635]: please check if it overlaps with auto generated structs in pb.go

Comment (Ramky-Infoblox) [reply example/feature_demo/demo_service.pb.gorm.go:635]: In order to remove overlapping of auto generated structs we can rename `XXXXXXListResponse` to any of the below options

1. `XXXXXXPIListResponse` (short form of PageInfo)
2. `XXXXXXPageInfoListResponse`  (little lengthy)
3. `XXXXXXCountListResponse` (specific to Count but in future if we want to use other fields of PageInfo then this name wouldn't be suitable)

File: example/feature_demo/demo_service.pb.gorm.go [hunk 2]
```diff
@@ -585,6 +585,58 @@ type IntPointORMWithAfterListFind interface {
 	AfterListFind(context.Context, *gorm1.DB, *[]IntPointORM, *query1.Filtering, *query1.Sorting, *query1.Pagination, *query1.FieldSelection) error
 }
 
+// DefaultListCountIntPoint executes a gorm list call with total record count
+func DefaultListCountIntPoint(ctx context.Context, db *gorm1.DB, f *query1.Filtering, s *query1.Sorting, p *query1.Pagination, fs *query1.FieldSelection) (*IntPointListResponse, error) {
```
[the remark this answers is not in the document]
Comment (Ramky-Infoblox) [reply example/feature_demo/demo_service.pb.gorm.go]: Sure

File: example/feature_demo/demo_service.pb.gorm.go (hunk 1 above)
Comment (Ramky-Infoblox) [reply example/feature_demo/demo_service.pb.gorm.go]: @chinmayb as discussed going ahead with 2 option

**Review by chinmayb — APPROVED** [review]

**Pull request merged**